ts/test/test-profiler.ts

```diff
@@ -534,18 +655,41 @@ describe('Profiler', () => {
          requestStub =
              sinon.stub(common.ServiceObject.prototype, 'request')
                  .onCall(0)
-                 .returns(new Promise(resolve => {
-                   resolve([requestProfileResponseBody, {statusCode: 200}]);
-                 }))
-                 .onCall(1)
-                 .returns(Promise.reject('Error uploading profile'));
+                 .callsArgWith(
+                     1, undefined, undefined,
+                     {statusCode: 409, body: {details: {retryDelay: '1000h'}}});
 
+         const profiler = new Profiler(testConfig);
+         profiler.timeProfiler = instance(mockTimeProfiler);
+         const delayMillis = await profiler.collectProfile();
+         assert.equal(7 * 24 * 60 * 60 * 1000, delayMillis);
```

Comment:
Would there be a downside to using `parse-duration` here? Or better, could you use `defaultConfig.maxBackoff...` here directly?

Reply:
I like that this test somewhat confirms that parseDuration is behaving as expected. So, losing that would be the downside.

Reply:
But, this is not a unit test for `parse-duration`. Unit tests should focus on the unit under test.

Reply:
True -- in tests in initConfig() I'll continue to specify the exact time (since if parseDuration fails there, then initConfig() cannot be working correctly either), but for here it may make sense to use parseDuration. 
Done.